.codeclimate.yml

```diff
@@ -0,0 +1,45 @@
+---
+exclude_paths:
+- ".git/"
+- "**.xml"
+- "**.yaml"
+- "**.yml"
+- "spec/"
+- "ci/"
+engines:
+  brakeman:
+    # very slow :sad_panda:
+    enabled: false
+  bundler-audit:
+    # requires Gemfile.lock
+    enabled: false
+  csslint:
+    enabled: false
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - ruby
+  eslint:
+    enabled: false
+    channel: "eslint-3"
```

Comment:
We probably will never have javascript and css in this repo so we can probably remove eslint and csslint engines from this list.